test/unit/app/models/editionable_worldwide_organisation_test.rb

```diff
@@ -179,7 +179,7 @@ class EditionableWorldwideOrganisationTest < ActiveSupport::TestCase
 
   test "should clone office and contact associations when new draft of published edition is created" do
     contact = create(:contact, translated_into: [:es])
-    published_worldwide_organisation = create(:editionable_worldwide_organisation, :published)
+    published_worldwide_organisation = create(:editionable_worldwide_organisation, :published, translated_into: [:es])
```

Comment:
Not sure if GDS likes to avoid red commits, but if so is it worth squashing this into the last commit?

Reply:
Seems like a good idea.  I've squashed them into one now.